Huizenjacht/src/main/java/be/pxl/huizenjacht/domain/House.java

```diff
@@ -1,104 +0,0 @@
-package be.pxl.huizenjacht.domain;
-
-import com.fasterxml.jackson.annotation.JsonProperty;
-import jakarta.validation.constraints.Min;
-import jakarta.validation.constraints.NotBlank;
-import jakarta.validation.constraints.NotNull;
-
-public class House {
-
-    @NotBlank

```

Comment:
Het is een goed idee om je eigen message hieraan mee te geven.
Bijvoorbeeld @NotBlank( message = "Code cannot be empty")